apps/vs-code-react/src/app/export/export.tsx

```diff
@@ -1,57 +1,29 @@
 // eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-import messages from '../../../../../libs/services/intl/src/compiled-lang/strings.json';
 import type { OutletContext } from '../../run-service';
 import type { RootState } from '../../state/store';
 import './export.less';
 import { Navigation } from './navigation/navigation';
 import { Text } from '@fluentui/react';
-import type { OnErrorFn } from '@formatjs/intl';
-import { useCallback } from 'react';
-import { IntlProvider } from 'react-intl';
-import { QueryClient, QueryClientProvider } from 'react-query';
 import { useSelector } from 'react-redux';
 import { Outlet, useOutletContext } from 'react-router-dom';
 
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      refetchInterval: false,
-      refetchIntervalInBackground: false,
-      refetchOnWindowFocus: false,
-      refetchOnReconnect: false,
-      refetchOnMount: true,
-    },
-  },
-});
-
 export const ExportApp: React.FC = () => {
-  const projectState = useSelector((state: RootState) => state.project);
   const workflowState = useSelector((state: RootState) => state.workflow);
 
-  const handleError: OnErrorFn = useCallback((err) => {
-    if (err.code !== 'MISSING_TRANSLATION') {
-      throw err;
-    }
-  }, []);
-
-  return projectState.initialized ? (
```

Comment:
We are already checking this initialized part in stateWrapper; we only navigate to this part if it is initialized